cli/tests/testdata/jsr/deps/main.ts

```diff
@@ -0,0 +1,3 @@
+import value from "jsr:@denotest/deps/mod.ts";
+
+console.log(value);
```

Comment:
How are these files getting served?

Reply:
This isn't part of the test registry which is in `testdata/jsr/registry`, so this is just a normal file in the testdata directory.